_posts/2019-11-18-meetup.md

```diff
@@ -0,0 +1,51 @@
+---
+title: DevLaunchers - world's most FUN experiment to help teenagers learn game development!
+author: kevin
+layout: post
+categories:
+- education
+- games
+- volunteering
+when: 2019-11-19T19:30:00-05:00
```

Comment:
Time is off by one hour. I guess this should be `-04:00`, @aaronj1335?

Reply:
Dang daylight savings time.

Reply:
Looks like it should be `-06:00`

https://time.is/Austin#time_zone

Reply:
https://github.com/austinjavascript/austinjavascript.github.io/pull/101